src/components/form/question/NumberInput.tsx

```diff
@@ -12,51 +12,55 @@ type Props = {
   min?: number
   id?: string
   className?: string
+  defaultValue?: string | number | null | undefined
 }
 
+forwardRef(function DebouncedInputWithForwardRef(
+  props: React.ComponentProps<typeof DebounceInput>,
+  ref: React.ForwardedRef<HTMLInputElement>
+) {
+  return <DebounceInput type="number" {...props} inputRef={ref} />
+})
+
 export default function NumberInput({
   unit,
   value = '',
   isMissing,
   setValue,
-  min = 0,
   className,
   id,
   ...props
 }: HTMLAttributes<HTMLInputElement> & Props) {
-  const locale = useLocale()
-
   const handleValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     const inputValue = event.target.value
+
     if (inputValue === '') {
       setValue(undefined)
     } else {
-      setValue(Number(inputValue))
+      setValue(unformatNumber(inputValue))
     }
   }
 
+  function unformatNumber(number: string) {
+    // Supprimer les séparateurs de milliers
+    return Number(number.replace(/[^0-9.-]+/g, ''))
+  }
```

Comment:
Should be static (e.g. helper or out of the component) to not re create function on each render

Reply:
Yes !